### ISSUE TITLE
plt.yscale and mplhep.histplot / histogram log scale on y?

### ISSUE THREAD

**Issue by ryuwd** [scikit-hep/mplhep#94]
Hi All,
Very glad to see there are people dedicated to making matplotlib a suitable plotting solution for HEP data.

I have a problem with setting a log scale on the y-axis when plotting a pre-binned histogram (uproot reads variable from root tree --> `np.histogram`)

Am I missing something or doing something wrong here?

```python
    plt.figure(figsize=(6,6))
    hep.histplot(histnom[0], histnom[1], label='Nominal')
    hep.histplot(histmisal[0], histmisal[1], label='Misaligned')
    plt.legend(fontsize='x-small')
    plt.gca().xaxis.set_minor_locator(AutoMinorLocator())
    plt.gca().yaxis.set_minor_locator(AutoMinorLocator())
    plt.title("Momentum resolution (deent)", fontsize=16)
    plt.xlabel("$p - p_{MC}$ [MeV]", fontsize=14)
    plt.ylabel("Tracks / %.1f MeV" % (bin_sz), fontsize=14)
    plt.yticks(fontsize=14)
    plt.xticks(fontsize=14)
    plt.yscale('log')

```

Here is the produced plot:
![ylogscale](https://user-images.githubusercontent.com/56410978/75926323-41aec480-5e62-11ea-8bb4-c75bd9b105ff.png)

Works without `plt.yscale`

**Comment by andrzejnovak** [comment]
Hmm, I should fix that, but you can get around it by setting ylim to a small non-zero number.

**Comment by ryuwd** [comment]
Ah, this sounds like a good idea - I will try this soon and let you know how it looks. 

Thanks!

**Comment by andrzejnovak** [comment]
@ryuwd Fixed in #150

**Comment by ryuwd** [comment]
Apologies I didn't follow up - but I did manage to get it working after your suggestion. Thanks for fixing it!
